requirements.txt

```diff
@@ -2,16 +2,14 @@ azure-storage-blob==12.8.1
 flake8==3.9.2
 python-magic==0.4.24
 requests-mock==1.9.3
-colorama
-importlib-metadata
-keyring
-numpy
-pandas
-pathlib
-Pillow
+colorama==0.4.4
+importlib-metadata==4.8.1
+keyring==23.1.0
```

Comment:
Not sure that these 3 are actually used. They don't show up in a search of the codebase.